NetKAN/Mk2Expansion.netkan

```diff
@@ -7,14 +7,14 @@
         { "name": "KlockheedMartian-Gimbal" },
         { "name": "ModuleManager", "min_version": "2.5.3" },
         { "name": "CommunityResourcePack" },
-        { "name": "InterstellarFuelSwitch" }
+        { "name": "InterstellarFuelSwitch-Core" }
     ],
     "suggests": [
 	{ "name": "FerramAerospaceResearch" }
     ],
     "install" : [
         {
-            "find"       : "Mk2Expansion",
+            "find"       : "GameData/Mk2Expansion",
```

Comment:
We don't need to use `"GameData/Mk2Expansion"`, as `find` will ignore `Mk2Expansion-1.6.06`